guix-science/packages/zotero.scm

```diff
@@ -0,0 +1,63 @@
+(define-module (guix-science packages zotero)
+  #:use-module (guix packages)
+  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module (gnu packages base)
+  #:use-module (guix download)
+  #:use-module (nonguix build-system binary)
+  #:use-module (gnu packages gcc)
+  #:use-module (gnu packages gtk)
+  #:use-module (gnu packages xorg)
+  #:use-module (gnu packages glib)
+  #:use-module (ice-9 regex))
+
+    (define-public zotero (package
+      (name "zotero")
+      (version "6.0.26")
+      (source (origin
+               (method url-fetch)
+               (uri (string-append
+                     "https://download.zotero.org/client/release/" version
+                     "/Zotero-" version "_linux-x86_64.tar.bz2"))
+               (sha256
+                (base32
+                 "0h1bizcqlk22h7nvhnyp3ppymv2hrk7133hgmp15hl3bvfzz7nh6"))))
+      (build-system binary-build-system)
+      (arguments
+       `(#:patchelf-plan `(("zotero-bin" ("gcc:lib"))
+                           ("libmozgtk.so" ("gtk+"))
+                           ("libxul.so" ("libx11" "dbus-glib" "libxt" "libgthread")))
+         #:install-plan '(("./" "/opt/zotero/"))
+         #:phases (modify-phases %standard-phases
```

Comment:
Consider breaking the line after `#:phases`.

Reply:
Done